qubole-aws-terraform-deployment/qubole-deployment/modules/account_integration/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "deployment_suffix" {
+}
+
+variable "qubole-aws-account-id" {
+  default = 805246085872
+}
+
+variable "qubole-external-id" {
+  default = "GI3DEMLIMRUWCZLC"
```

Comment:
Did you intend to leave the values in the variables.tf?

Reply:
Noted. removed defaults for external id, the region as well as qubole-defloc name. This will force users to review these variables and apply the correct ones as per their Qubole account